plumbing/api.go

```diff
@@ -112,6 +113,15 @@ func (api *API) ChainLs(ctx context.Context) <-chan interface{} {
 	return api.chain.BlockHistory(ctx, api.chain.Head())
 }
 
+// ChainGetActorFromLatestState returns an actor from the latest state on the chain
+func (api *API) ChainGetActorFromLatestState(ctx context.Context, addr address.Address) (*actor.Actor, error) {
```

Comment:
`ActorGet` would be a more consistent and concise name. Apologies that I suggested this one. Beyond that, I have 3 concerns:

1. This is the only method in API that is not a simple pass-through.
2. This isn't tested. This is probably related to the above.
3. We're importing `chain.ReadStore` instead of an interface off it. This made sense when we were simply exposing its methods, but less so now that we are deriving functionality from it. Writing a test for this method is slightly non-trivial which suggests this is a problem.

The real question is whether this should be made a plumbing/something/Something-er like config and messages. It seems too much structure for such a simple thing, but then this seems a bit too little.

thoughts @phritz?